learn/configuration/instance_options.md

```diff
@@ -169,16 +171,53 @@ Sets the directory where Meilisearch will create dump files.
 
 ### Import dump
 
+::: warning
+This is a CLI flag and does not take any values. Assigning a value will throw an error.
+:::
+
 **Environment variable**: N/A
 **CLI option**: `--import-dump`
-**Default value**: `none`
-**Expected value**: a filepath pointing to a `.dump` file
+**Default**: Disabled
 
-Imports the dump file located at the specified path. Path must point to a `.dump` file.
+Replaces `data.ms` with dump data and starts the server using the provided dump. Path must point to a `.dump` file.
```

Comment:
This explanation seems misleading to me. My understanding is that if `data.ms` already exists, Meilisearch will throw an error and abort launch (unless `--ignore-missing-dump` is given). So `data.ms` will not be replaced by default.

I prefer the present explanation.

```suggestion
Imports the dump file located at the specified path. Path must point to a `.dump` file. If a database already exists, Meilisearch will throw an error and abort launch.
```